.github/workflows/test.yml

```diff
@@ -2,28 +2,27 @@ name: Test
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node-version: [20.x]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install
-    - run: npm test
-    - name: Upload HTML report(backstop data)
-      if: ${{ always() }}
-      uses: actions/upload-artifact@v4
-      with:
-        name: report
-        path: backstop_data
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
```

Comment:
Consider updating the `actions/setup-node` action to a more recent version, such as `v3`, to ensure compatibility and access to the latest features and security updates.